app/controllers/prompts_controller.rb

```diff
@@ -158,4 +159,45 @@ def destroy
     end
   end
 
+  private
+
+  def prompt_params
+    params.require(:prompt).permit(
+      :id,
```

Comment:
This might be unnecessary -- I couldn't find it in the relevant forms, and the tests seemed to pass without it.